Mid.html

```diff
@@ -0,0 +1,108 @@
+<!Doctype html>
+<html>
+    <head lan="en">
+        <link rel="stylesheet" type="text/css" href="Mid.css">
+        <title>MiD</title>
+    </head>
+    <body>
+	<div class="resume">
+         <div class = "externalDiv">
+      <div class = "internalDiv">
+	    <p id="header">TEAM <span style="color:red">RED</span></p> 
+        <p id="subHeader">Software Developers</p>
+      </div>
+        </div>
+          <div class="container">
+              <div class="Left">
+              <h3 id="heading3">PROFILE</h3>
+              <p id="para">To obtain a position that will enable me to use my strong organizational skills, award-winning educational background, and ability to work well with people.To obtain a position that will enable me to use my strong organizational skills, award-winning background, and ability to work well with people</p>
+        </div>
+              <div class="mid">
+                  <img src="images/img_avatar.png" alt="Avatar" >
+              </div>
+              <div class="Right">
+                  <p>101,Viman Nagar,pune   <img src="images/home.png" alt="home" class="smallimage"></p>
+                  <p>gautamraaj@gmail.com   <img src="images/mail2.png" alt="Mail" class="smallimage"></p>
+                  <p>personaldeatils.com  <img src="images/lap.jpg" alt="Mail" class="smallimage"></p>
+                  <p>8122171333   <img src="images/phone.png" alt="Mail" class="smallimage"></p>
+              </div>
+        </div>
+	<div class="outermost">
+	<hr size="3px" noshade style="">
+	<div>
+	<span class="whites"><b>EDUCATION</b></span>
+	<span class="peaches"><b>SKILLS</b></span>
+	<span class="whites" style="top:-28px; left:-20px; float:right;"><b>EXPERTISE</b></span>
+	</div>
+	<div>
+	<div align="left">
+	<p> Associate degree in graphic design<br>University of illinois, 2002</p>
+	<p> Bachelor degree in interior design<br>University of illinois, 2004</p>
+	<p> Certificate in online marketing<br>University of illinois, 2006</p>		
+	</div>
+	<div align="center" style="position:relative; top:-160px; left:10px;">
+	<p><b>MAC/PC SYSTEMS</b></p>
+	<p><b>MICROSOFT WORD</b></p>
+	<p><b>POWERPOINT</b></p>
+	<p><b>BLOGGING</b></p>
+	<p><b>PHOTOSHOP CS</b></p>
+	</div>
+	<div  align="right" style="position:relative; top:-330px; left:-25px;">
+	<p><b>PROJECT DESIGN</b></p>
+	<p><b>SOCIAL MEDIA</b></p>
+	<p><b>QUALITY CONTROL</b></p>
+	<p><b>REPORT WRITING</b></p>
+	<p><b>WEB DESIGN</b></p>
+	</div>
+	</div>
+	<br>
+	<br>
+	<hr size="3px" noshade style="position:relative; top:-350px;">
+	<div>
+	<span class="peaches" style="padding-left:42px; padding-right:42px; left:41%; position:relative; top:-370px;"><b>EXPERIENCE</b></span>
+	</div>
+	</div>
+	<div class="full">
+    <div class="entire">
+        <div class="content">
+            <p class="h2class"><b style="font-size:130%">UI Developer</b></p>
```

Comment:
what is this h2class

use proper naming

Reply:
Done.